### PR TITLE
hotfix for #127

### PR DESCRIPTION
**Pull request by henderkes**
self explanatory

### CODE REVIEW

**Review by Leats — APPROVED** [review]
All tests pass! Thank you!

File: backend/flask_api.py [hunk 1]
```diff
@@ -55,19 +55,17 @@ def add_project():
...
-                    <body>Successful files: """ + ', '.join(e for e in successful_files) + '<br />' + """
-                    Unsuccessful files: """ + ', '.join(e for e in unsuccessful_files) + """
-                    </body>"""
+        return make_response('Successful files: ' +
+                             ', '.join(e for e in successful_files) +
+                             '<br />' + " Unsuccessful files: " +
+                             ', '.join(e for e in unsuccessful_files))
 
-    else:  # no files attached
+    elif request.content_type == 'application/json' or request.content_type == 'application/json5':
```
Comment (Drakulix) [inline backend/flask_api.py:68]: Oh no, this is ugly. This mean 'application.json' can pass, if the contents are actually 'application/json5' (vise versa is not a big deal and allowed).

**Review by Drakulix — COMMENTED** [review]
Why was that merged?

File: backend/flask_api.py (hunk 1 above)
Comment (henderkes) [reply backend/flask_api.py:68]: with 'application/json' set `if request.json` will be the case, the `elif request.data:` will not be executed. The problem you described can therefore not happen. json5 content in request.json will result in an ApiException being raised

**Comment by henderkes** [comment]
I restored the branch to make it prettier down the road, I actually don't like the design this way myself (double if check). this also references issue #141 

@Drakulix you are welcome to revert the merge if you are too unhappy with the current code, it should however not present a functionality issue

**Pull request merged**